tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -686,12 +678,17 @@ private void cleanUp() {
       if (amRpcClient != null) {
         amRpcClient.finishApplication();
       }
-      FileSystem fs = FileSystem.get(hdfsConf);
-      if (appResourcesPath != null && fs.exists(appResourcesPath)) {
-        fs.delete(appResourcesPath, true);
-      }
     } catch (IOException | YarnException e) {
-      LOG.error("Failed to clean up temporary files :" + appResourcesPath, e);
+      LOG.error("Failed to finish application.", e);
+    } finally {
+      try {
```

Comment:
can we create a util method for this `try-catch` blob? This looks the same as the one above

Reply:
hmm...how do we refactor a try-catch block? we'd have to pass the block as an argument to the util function.

Reply:
i mean having a 
```
deleteFile(path, msg) {
   try {
   } catch {
      Log.error(msg)
   }
}
```

Reply:
Ah gotcha...fixed